Detections/CommonSecurityLog/DiscordCDNRiskyDownload.yaml

```diff
@@ -0,0 +1,51 @@
+id: 010bd98c-a6be-498c-bdcd-502308c0fdae
+name: Discord CDN Risky File Download
+description: |
+  'Identifies  callouts to Discord CDN addresses for risky file extensions. This detection will trigger when a callout for a risky file
+  is made to a discord server that has only been seen once in your environment. Unique discord servers are identified using the server ID
+  that is included in the request URL (DiscordServerId in query). Discord CDN has been used in multiple campaigns to download additional payloads'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Zscaler
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1d
+queryPeriod: 7d
```

Comment:
If we go with 7 day query period, but run every day, we will get duplicates.  Also, 7 days of data for commonsecuritylog may be large and unnecessarily resource intensive.  Change to 1d.